internal/extramanifest/extramanifest.go

```diff
@@ -79,78 +109,31 @@ func (h *EMHandler) ExportExtraManifestToDir(ctx context.Context, extraManifestC
 	return nil
 }
 
-// ApplyExtraManifestsFromDir applies the extra manifests from the given directory
-func (h *EMHandler) ApplyExtraManifestsFromDir(ctx context.Context, fromDir string) error {
-	manifestDir := filepath.Join(fromDir, extraManifestPath)
-	manifestYamls, err := os.ReadDir(manifestDir)
+// ApplyExtraManifests applies the extra manifests from the preserved extra manifests directory
+func (h *EMHandler) ApplyExtraManifests(ctx context.Context, executor ops.Execute) error {
```

Comment:
I wonder, we are doing it in post pivot phase too. 
applying all manifests from /opt/openshift/extra_manifests folder. Why we need same logic twice?

Reply:
I believe this would be the day2 bits that we'd want applied after the system is stable, similar to the day2 policies for ztp gitops, vs those that need to be applied during the earlier stage as part of bringing the system back up

Reply:
Hmm, 
Maybe we need to delete https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/postpivot/postpivot.go#L203 and use code here